snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml

```diff
@@ -165,3 +165,372 @@ metrics:
+  # * Clients can use a virtual server as an entrypoint to an application service that spans multiple machines.
+  # Learn more: https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-basics-11-6-0.html
+
+  # Virtual servers.
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.3375.2.2.10.1.1
+      name: ltmVirtualServNumber
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.1.2
+      name: ltmVirtualServTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.9
+        name: ltmVirtualServEnabled
+      - OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.10
+        name: ltmVirtualServConnLimit
+    metric_tags:
+      - tag: server
+        column:
+          OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.1
+          name: ltmVirtualServName
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.2.3
+      name: ltmVirtualServStatTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.5
+        name: ltmVirtualServStatNoNodesErrors
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.6
+        name: ltmVirtualServStatClientPktsIn
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.7
+        name: ltmVirtualServStatClientBytesIn
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.8
+        name: ltmVirtualServStatClientPktsOut
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.9
+        name: ltmVirtualServStatClientBytesOut
+    metric_tags:
+      - tag: server
+        column:
+          OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.1
+          name: ltmVirtualServStatName
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.2.3
+      name: ltmVirtualServStatTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.11
+        name: ltmVirtualServStatClientTotConns
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.27
+        name: ltmVirtualServStatTotRequests
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.40
+        name: ltmVirtualServStatClientEvictedConns
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.41
+        name: ltmVirtualServStatClientSlowKilled
+    forced_type: monotonic_count
+    metric_tags:
+      - tag: server
+        column:
+          OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.1
+          name: ltmVirtualServStatName
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.2.3
+      name: ltmVirtualServStatTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.12
+        name: ltmVirtualServStatClientCurConns
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.31
+        name: ltmVirtualServStatVsUsageRatio5s
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.32
+        name: ltmVirtualServStatVsUsageRatio1m
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.33
+        name: ltmVirtualServStatVsUsageRatio5m
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.34
+        name: ltmVirtualServStatCurrentConnsPerSec
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.35
+        name: ltmVirtualServStatDurationRateExceeded
+    forced_type: gauge
```

Comment:
Because the table you linked to is intended to submit rates 😅

SNMP metric types there are correct (`Counter64`) and we somehow don't support `forced_type: rate` yet, so I didn't add an explicit `forced_type` there. But I probably should because I agree it's confusing.